docs/construct.md

```diff
@@ -0,0 +1,237 @@
+# Construct Sparse Arrays
+
+## From coordinates and data
+
+You can construct [COO][sparse.COO] arrays from coordinates and value data.
+
+The `cords` parameter contains the indices where the data is nonzero,
+and the `data` parameter contains the data corresponding to those indices.
+For example, the following code will generate a $5 \times 5$ diagonal
+matrix:
+
+```python
+
+   >>> import sparse
```

Comment:
Is the indentation in these cases necessary? If not, I'd remove it.

Reply:
Do you mean on all the code boxes? (all files)?

Reply:
Yes.